core/block_animations.ts

```diff
@@ -26,7 +26,7 @@ interface CloneRect {
 }
 
 /** PID of disconnect UI animation.  There can only be one at a time. */
-let disconnectPid: AnyDuringMigration = 0;
+let disconnectPid: ReturnType<typeof setTimeout>|null = null;
```

Comment:
It feels weird to change the type of this to conform to node stuff instead of browser stuff. But I guess these types are only seen internally, so it should be fine?

Reply:
It's both, it'll be browser types in a browser and Node types in Node. But yeah I don't think it should be a big deal, this field should probably be private anyway?